script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_6_2.txt

```diff
@@ -136,42 +136,42 @@ His mastery of the billhook is also lost.
 The vampire swings his billhook haphazardly as if to shake off the fear in front of him. But Arcueid is not going to let that disgraceful opening pass by.
 <Page38>
 "It's cold――― It's cold――― This place, is cold―――!"
-The vampire jumps as he gives off a scream.
-With the last of his blood, Vlov jumps to the back. No, he jumps above.
+The vampire leaps as he gives off a scream.
```

Comment:
The vampire leaps as he lets out a scream.

(gives off is to emanate, or imply)